src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
I might be missing something but why can't the code be something like:

    mudlet = mudlet or {}
    if type(mudlet.supports) == nil then
        mudlet.supports = {}
    end
    mudlet.supports["coroutines"] = true